app/themes/skins/FormFieldOwnSkin.scss

```diff
@@ -31,6 +31,14 @@
     border-radius: 8px;
 
     margin-top: -17px;
+    background-color: transparent;
```

Comment:
is this something forgotten to delete?

Reply:
No, it's part of the trick, otherwise the background keeps overlapping.

Reply:
@leivaburto 
I deleted `background-color: transparent;` and it still works on all cases i can think of.
Can you explain in which case do you need this?

logically `-webkit-transition-delay: 99999s;` delays the `input:-webkit-autofill` transition for `99999s` so i am not sure why `background-color: transparent;` is needed at all.

Reply:
You are right! Must have been something that helped me in a partial solution, but now is no longer needed as you said.